src/script/components/Image.tsx

```diff
@@ -17,83 +17,131 @@
  *
  */
 
-import React, {useEffect, useState} from 'react';
+import React, {useEffect, useRef, useState} from 'react';
 
-import cx from 'classnames';
+import {CSSObject} from '@emotion/react';
 import {container} from 'tsyringe';
 
 import {useKoSubscribableChildren} from 'Util/ComponentUtil';
 
 import {RestrictedImage} from './asset/RestrictedImage';
+import {AssetUrl, useAssetTransfer} from './MessagesList/Message/ContentMessage/asset/useAssetTransfer';
 import {InViewport} from './utils/InViewport';
 
 import {AssetRemoteData} from '../assets/AssetRemoteData';
-import {AssetRepository} from '../assets/AssetRepository';
+import {Config} from '../Config';
+import {MediumImage} from '../entity/message/MediumImage';
 import {TeamState} from '../team/TeamState';
 
-export interface ImageProps extends React.HTMLProps<HTMLDivElement> {
-  aspectRatio?: number;
-  width?: string;
-  asset: AssetRemoteData;
-  assetRepository?: AssetRepository;
-  click?: (asset: AssetRemoteData, event: React.MouseEvent) => void;
+const imageWrapperStyle = {
+  width: '100%',
+};
```

Comment:
I don't feel strongly about it, but I think it's missing
```javascript
  backgroundColor: 'var(--foreground-fade-8)',
  ```
from the previous imlementation, no?